.gitignore

```diff
@@ -72,3 +72,6 @@ typings/
 
 # Test cafe reports
 report*.json
+
+# JetBrains IDE config
+.idea/
```

Comment:
I think this convo came up before actually but it might be better to keep IDE specific configs in your local `.gitignore` file. It feels more the responsibility of the individual developer than the project, as we can't know what every developer's preferred IDE choice is.

Reply:
See

https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/pull/437#discussion_r412227978

https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/pull/787

But it's up to Adam and Nev now, innit?

Reply:
I agree with this, I'll change it